recipe/meta.yaml

```diff
@@ -33,19 +31,28 @@ requirements:
   host:
     - glib
 
-
 test:
   commands:
-    - test -f ${PREFIX}/lib/libgts.so  # [linux]
-    - test -f ${PREFIX}/lib/libgts.dylib  # [osx]
+    - test -f ${PREFIX}/lib/libgts.so                     # [linux]
+    - test -f ${PREFIX}/lib/libgts.dylib                  # [osx]
     - if not exist %LIBRARY_PREFIX%\\bin\\gts.dll exit 1  # [win]
 
 about:
   home: http://gts.sourceforge.net/
-  license: LGPL-2.0-or-later
+  license: GPL verision 2
```

Comment:
This is not a valid SPDX license identifier, plus it is misspelled.

Reply:
I fixed the typo.  I wasn't aware that the `license` field has to conform with the SPDX license identifier.  My understanding was that `license` is a free format field, and `license_family` has to be one of the strings listed here: https://github.com/conda/conda-verify/blob/master/conda_verify/constants.py#L8